data.js

```diff
@@ -287,4 +287,10 @@ var data = [
     'text': ['der politische Gegner als \"Ergebnis von Sodomie und Inzucht\" bezeichnet'],
     'src': ['https://www.noz.de/deutschland-welt/politik/artikel/955751/wie-rechts-ist-die-afd']
   }
+   {
+    'forename': 'Heiko',
+    'name': 'Maas',
+    'text': ['der ein Gesetz erarbeitet hat, dass das Grundrecht auf Presse- und Meinungsfreiheit massiv beschädigen könnte'],
```

Comment:
```erarbeitet hat, dass das``` "dass" ist an dieser stelle leider nicht korrekt ;)
Änder das doch einfach in ```erarbeitet hat, welches das```.